storybook/stories/Examples/RadialBarChart.stories.tsx

```diff
@@ -29,12 +42,13 @@ export const SimpleRadialBarChart = {
 };
 
 export const RadialBarWithColors = {
-  render: (args: Record<string, any>) => {
+  render: (args: StorybookArgs, context: StoryContext) => {
     return (
       <RadialBarChart {...args}>
         <RadialBar dataKey="pv" />
         <Legend />
         <Tooltip />
+        <Customized component={<RechartsHookInspector portalRef={context.portalRef} />} />
```

Comment:
I was searching for an API that will allow me to attach the inspector everywhere without changing the stories but I couldn't find a way. This appears to be the minimum possible change unless we want to rewrite the AST on the fly which I would rather avoid.

Reply:
This will break soon though right?

Reply:
The breaking part is that we're going to remove the `props` and `state`, we don't need to remove the `Customized` component itself. It will be a "just render component" useless appendix but it can stay there for a while. The inspector doesn't use props or state so it should be fine.

Reply:
Hm interesting, I thing we were removing it entirely. I guess no harm in it being there but it might confuse people if it's kept in 3.0

Reply:
We can do that too and migrate all the usages with codemod.